Unity3D/Assets/RosSharp/Scripts/ImagePublisher.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace RosSharp.RosBridgeClient
+{
+	[RequireComponent(typeof(RosConnector))]
+	public class ImagePublisher : MonoBehaviour {
+		
+		public Camera cameraToPublish;
+		public string baseTopic = "/image_raw";
```

Comment:
Please set default value to "/image_raw/compressed" and remove +"/compressed" in line 29 below.